migrations/20210412140907-remove-notifications-unique-type.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      DROP INDEX IF EXISTS "Notifications_type_CollectiveId_UserId";
```

Comment:
Removed this index as it's perfectly valid to define multiple webhooks for the same event (eg. one for Discord, one for Slack, one for your website, etc).